service/controller/release/resource/label/create.go

```diff
@@ -0,0 +1,70 @@
+package status
```

Comment:
```suggestion
package label
```

This looks wrong.

Reply:
Thanks. I should also wire the resource.

Reply:
Updated the package name and wired the resource.